lib/galaxy/tool_util/ontologies/ontology_data.py

```diff
@@ -0,0 +1,115 @@
+from typing import (
+    cast,
+    Dict,
+    List,
+    NamedTuple,
+    Optional,
+    Tuple,
+)
+
+import importlib_resources
+
+from galaxy.tool_util.biotools import BiotoolsMetadataSource
+from galaxy.tool_util.parser import ToolSource
```

Comment:
```suggestion
from galaxy.tool_util.biotools import BiotoolsMetadataSource
from galaxy.tool_util.parser import ToolSource
from galaxy.util.resources import files
```

We have a little wrapper there that deals with importlib.resources not being in python 3.9